datasets/bomen/paspoortdata/v1.0.0.json

```diff
@@ -0,0 +1,443 @@
+            "reasonsNonPublic": [
+              "5.2 4: Zwaarwegend belang: persoonlijke beleidsopvattingen (bevat milieu-informatie)" ]
+            },
+            "kroondiameterklasseActueel": {
+                "type": "string",
+                "description": "Diameter van de kroon van de boom in meters ingedeeld in vaste klassen.",
+                "auth": "FP/MDW",
+            "reasonsNonPublic": [
+              "5.2 4: Zwaarwegend belang: persoonlijke beleidsopvattingen (bevat milieu-informatie)" ]
+            },
+            "kroondiameterklasseEindbeeld": {
+                "type": "string",
+                "description": "Diameter van de kroon van het eindbeeld van de boom in meters ingedeeld in vaste klassen.",
+                "auth": "FP/MDW",
+            "reasonsNonPublic": [
+              "5.2 4: Zwaarwegend belang: persoonlijke beleidsopvattingen (bevat milieu-informatie)" ]
+            },
+            "kroonvolume": {
+                "type": "integer",
+                "description": "Volume van de boomkroon in kubieke meters",
+                "auth": "FP/MDW",
+            "reasonsNonPublic": [
+              "5.2 4: Zwaarwegend belang: persoonlijke beleidsopvattingen (bevat milieu-informatie)" ]
+            },
+            "kweker": {
+                "type": "string",
+                "description": "Kweker van het vegetatieobject.",
+                "auth": "FP/MDW",
+            "reasonsNonPublic": [
+              "5.2 4: Zwaarwegend belang: persoonlijke beleidsopvattingen (bevat milieu-informatie)" ]
+            },
+            "leeftijd": {
+                "type": "string",
+                "description": "Leeftijd van het beheerobject in jaren.",
+                "auth": "FP/MDW",
+            "reasonsNonPublic": [
+              "5.2 4: Zwaarwegend belang: persoonlijke beleidsopvattingen (bevat milieu-informatie)" ]
+            },
+            "leverancier": {
+                "type": "string",
+                "description": "Leverancier van het beheerobject.",
+                "auth": "FP/MDW",
+            "reasonsNonPublic": [
+              "5.2 4: Zwaarwegend belang: persoonlijke beleidsopvattingen (bevat milieu-informatie)" ]
+            },
+            "ligging": {
+                "type": "string",
+                "description": "Aanduiding van de ligging van het beheerobject binnen of buiten de bebouwde kom.",
+                "auth": "FP/MDW",
+            "reasonsNonPublic": [
+              "5.2 4: Zwaarwegend belang: persoonlijke beleidsopvattingen (bevat milieu-informatie)" ]
+            },
+            "objectnaam": {
+                "type": "string",
+                "description": "Naam van het beheerobject.",
+                "auth": "FP/MDW",
+            "reasonsNonPublic": [
+              "5.2 4: Zwaarwegend belang: persoonlijke beleidsopvattingen (bevat milieu-informatie)" ]
+            },
+            "objectnummer": {
+                "type": "string",
```

Comment:
De naam suggereert eerder een integer of number.

Reply:
aangepast